iis-wf/iis-wf-import/src/main/java/eu/dnetlib/iis/wf/importer/infospace/ImportInformationSpaceJob.java

```diff
@@ -0,0 +1,404 @@
+package eu.dnetlib.iis.wf.importer.infospace;
```

Comment:
Yes, I have run it on a full set of data and it worked without any problems.

Initially I have considered splitting it into dedicated jobs, even on a more fine-grained level, but it turned out some of the input datastores are being subject for more than one output (e.g. datasets) so I decided to do this with one go. We could do the distinction e.g. between the entities and relations, as you have proposed, but since we are not dealing with any memory/performance related issues I would prefer to keep it compact.